libraries/ms-drivers/test/test_pcf8523_rtc.c

```diff
@@ -0,0 +1,76 @@
+#include "delay.h"
+#include "gpio.h"
+#include "log.h"
+#include "pcf8523_rtc.h"
+#include "pcf8523_rtc_defs.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#define TEST_I2C_PORT I2C_PORT_2
+
+#define TEST_CONFIG_PIN_I2C_SCL \
+  { GPIO_PORT_B, 10 }
+#define TEST_CONFIG_PIN_I2C_SDA \
+  { GPIO_PORT_B, 11 }
+
+I2CSettings i2c = {
+  .speed = I2C_SPEED_FAST,         //
+  .sda = TEST_CONFIG_PIN_I2C_SDA,  //
+  .scl = TEST_CONFIG_PIN_I2C_SCL,  //
+};

```

Comment:
This should be static+prefixed with `s_`, or just a local variable in `setup_test`.